core/src/main/java/org/dcache/nfs/vfs/PseudoFs.java

```diff
@@ -143,6 +143,10 @@ public int access(Inode inode, int mode) throws IOException {
     public Inode create(Inode parent, Stat.Type type, String path, Subject subject, int mode) throws IOException {
         Subject effectiveSubject = checkAccess(parent, ACE4_ADD_FILE);
 
+        if (subject != null && Subjects.isRoot(_subject)) {
```

Comment:
Almost there :). You have to check effective subject being root as it already passed export ceck and mapped to nobody, if needed (e.g squash_root).